config/spring19_seminar/cola_var/bert_target_tasks.conf

```diff
@@ -0,0 +1,25 @@
+//  This config loads a model pretrained with bert.conf (currently in the cola_var branch) and does target-task training
```

Comment:
Use probing somewhere in the name or the comment, since this config seems to be for probing-style experiments with frozen BERT.